fem/coefficient.cpp

```diff
@@ -28,6 +28,52 @@ double PWConstCoefficient::Eval(ElementTransformation & T,
    return (constants(att-1));
 }
 
+void PWCoefficient::InitMap(const Array<int> & attr,
+                            const Array<Coefficient*> & coefs)
+{
+   MFEM_ASSERT(attr.Size() == coefs.Size(),
```

Comment:
When there is no performance impact, I prefer `MFEM_VERIFY` over `MFEM_ASSERT`, and this seems to be the case for all the new `MFEM_ASSERT` calls in this file.

Reply:
Good point. I'll make this change